pytest_repo_health/fixtures/github.py

```diff
@@ -15,19 +17,28 @@
 
 URL_PATTERN = r"github.com[/:](?P<org_name>[^/]+)/(?P<repo_name>[^/]+).git"
 
+logger = logging.getLogger(__name__)
+
+GITHUB_TOKEN_MISSING_ERROR_MESSAGE = "Error. No GITHUB_TOKEN environment variable found. To use any of the GitHub " \
+                                     "fixtures, you must set the GITHUB_TOKEN environment variable to contain a " \
+                                     "GitHub personal access token."
+
 
 @pytest.fixture(scope="session")
 def github_client():
     """
     A fixture to initialize a GitHub API client, using the personal access token
     obtained from the GITHUB_TOKEN environment variable.
     """
+    github_client_container = namedtuple('github_client', 'object message')
```

Comment:
Stylistic nit-picks: the type name of a named tuple is normally formatted as a class name and also used as the variable to assign the result to, so something like `GitHubClient = namedtuple('GitHubClient', 'object message')`.